core/templates/dev/head/components/score_ring/score_ring_directive.html

```diff
@@ -0,0 +1,39 @@
+<svg class="score-ring"
+     width="250"
+     height="250">
+  <text x="78" y="130" fill="black"> <[getScore()]>%</text>
+  <circle class="score-outer-ring__circle"
+          stroke="#F4CEBA"
+          stroke-width="10"
+          fill="transparent"
+          r="100"
+          cx="125"
+          cy="125">
+  </circle>
+  <circle class="score-ring__circle"
+          stroke="#D95C0C"
+          stroke-width="10"
+          fill="transparent"
+          r="100"
+          cx="125"
+          cy="125">
+  </circle>
+</svg>
+<style>
+.score-ring__circle {
```

Comment:
Typically I think our convention is .score-ring-inner-circle, .score-ring-outer-circle (i.e. all kebab-case)

Reply:
Done